app/views/companies/show.html.haml

```diff
@@ -29,15 +29,16 @@
 
 
         .address
```

Comment:
Once the 'addresses...' epic has settled down, we should really refactor this part of the view into a partial and use helpers to get some of this logic out of the view.  (Or perhaps do this as part of the 'addresses...' epic work)

Reply:
This will be refactored as part of the epic work.